File: stavor/src/main/assets/www/modules/simulator/components/Mission.js
var Mission = function () {
	this._id = 0;
	this.name = "CustomMission";
	this.description = "";
	this.duration = 600000.0;
	this.step = 60.0;
	this.initial_date = new SimDate();
	this.initial_mass = 2000.0;
	this.initial_orbit = new KeplerianOrbit();
}

File: stavor/src/main/java/cs/si/stavor/web/Mission.java
package cs.si.stavor.web;

import java.io.Serializable;

/**
 * Created by xgibert on 17/02/2015.
 */
public class Mission implements Serializable {
    public String name = "CustomMission";
    public String description = "";
    public double duration = 600000.0;
    public double step = 60.0;
    public int attitude_provider = 0;
    public SimDate initial_date = new SimDate();
    public double initial_mass = 2000.0;
    public KeplerianOrbit initial_orbit = new KeplerianOrbit();
}
